### PR TITLE
Rename functions called run to avoid collision with stdlib

### PR DESCRIPTION
**Pull request by pakoito**
Title

### CODE REVIEW

File: modules/mtl/arrow-mtl-data/src/main/kotlin/arrow/mtl/Kleisli.kt [hunk 1]
```diff
@@ -15,8 +15,11 @@ import arrow.typeclasses.Monad
  */
 typealias KleisliFun<F, D, A> = (D) -> Kind<F, A>
 
+@Deprecated("Kleisli#run can be easily mistaken with the standard library extension function", ReplaceWith("runK(d)"))
 fun <F, D, A> KleisliOf<F, D, A>.run(d: D): Kind<F, A> = fix().run(d)
 
+fun <F, D, A> KleisliOf<F, D, A>.runK(d: D): Kind<F, A> = fix().run(d)
```
Comment (raulraja) [inline modules/mtl/arrow-mtl-data/src/main/kotlin/arrow/mtl/Kleisli.kt]: can we make this `fun operator invoke`?

File: modules/streams/arrow-streams/src/test/kotlin/arrow/streams/internal/FreeCTest.kt [hunk 2]
```diff
@@ -236,7 +236,7 @@ class FreeCTest : UnitSpec() {
     "Running a FlatMapped value using Either" {
       forAll(Gen.string()) { s ->
         FreeC.FlatMapped(FreeC.just("")) { FreeC.just<EitherPartialOf<Throwable>, String>(s) }
-          .run(Either.monadError()) == Right(Some(s))
+          .runFC(Either.monadError()) == Right(Some(s))
```
Comment (raulraja) [inline modules/streams/arrow-streams/src/test/kotlin/arrow/streams/internal/FreeCTest.kt]: we could also use invoke in these cases for Free. I think this is inline that both Free and Kleisli represent functions that you can apply to evaluate the structure.

File: modules/mtl/arrow-mtl-data/src/main/kotlin/arrow/mtl/Kleisli.kt (hunk 1 above)
Comment (pakoito) [reply modules/mtl/arrow-mtl-data/src/main/kotlin/arrow/mtl/Kleisli.kt]: I'd rather we didn't, at least for this first version. Automatic replacement can cause syntactic errors.

**Comment by pakoito** [comment]
https://travis-ci.org/arrow-kt/arrow/jobs/579540609#L41885

**Comment by rachelcarmena** [comment]
Hi! Libraries have been moved to the new repositories. Please, find the steps here: https://github.com/arrow-kt/arrow/blob/master/docs/move-to-multi-repo/how-to-move-a-pr.md and let me know if I can help you to move this PR to the correspondent repository. Thanks for your collaboration!!

**Pull request closed without merging**